src/config.coffee

```diff
@@ -7,38 +7,38 @@ checkInt = (s) ->
 		return
 	return i
 
-checkValidKey = (s) ->
+checkString = (s) ->
 	# Make sure `s` exists and is not an empty string, or 'null'.
-	if !s? or s == 'null' or s == ''
+	if !s? or s == 'null' or or s == 'undefined' or s == ''
```

Comment:
is "undefined" also a possible string? 

I'd also prefer we have some sort of explanation that this is due to a bug in OS or something, but that's up to you.

LGTM otherwise.